### ISSUE TITLE
'NoneType' object has not attribute 'strip'

### ISSUE THREAD

**Issue by ryanvillarreal** [ShawnDEvans/smbmap#28]
When executing smbmap's File Content Search with the -F command.  I am getting returned a 'NoneType' object has no attribute 'strip'

Command to reproduce:
`python smbmap.py -d DOMAIN -u USER -p PASS --host-file ~/Desktop/smb.txt -F '[1-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9][0-9][0-9]' `

Output produced: 
`'NoneType'  object has no attribute 'strip' \
[!] Job creation failed on host: x.x.x.x  \
[+] Grabbing search results, be patient, share drives tend to be big ...  \
[+] All jobs complete  
`

**Comment by ShawnDEvans** [comment]
Ok, this only took a full year, however I revamped the file search feature. I need to test it across hosts, but it no longer uses 'findstr' to search. I've instead opted for a PowerShell solution. I may add an option down the road so you can pick your preference, but 'findstr' didn't perform as expected when a non-default search path was specified. I think it had to do with how strings needed to be escaped. It's a convoluted solutions at best. PowerShell  Start-Process is used to launch a 'cmd' process, so the file search can run in the background, while the initial PowerShell Start-Process command exits. This way the search can be launched on a number of hosts, and periodically checked on to see if it's finished up. 

TL;DR - File search was revamped and should (hopefully) have less bugs than before.
